.github/workflows/deploy.yml

```diff
@@ -8,12 +8,56 @@ on:
     branches: [ master ]
 
 jobs:
-  deploy:
-    name: Deploy Demo App
+  build_browser_demo:
+    name: Build Browser Demo
+    runs-on: ubuntu-latest
+    env:
+        AWS_DEFAULT_REGION: us-east-1
+        AWS_DEFAULT_OUTPUT: text
+        AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY}}
+        AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install SAM CLI
+        run: script/github-action-awscli-installation
+      - name: Build Browser Demo
+        run: script/build-browser-demo
+  deploy_alpha_canary:
```

Comment:
Each of the deploy* job needs to have a "needs" attribute for the `build_browser_demo:` job to finish before the deploy job runs.

Reference: https://github.com/aws/amazon-chime-sdk-js/blob/master/.github/workflows/browser-compatibility-test.yml#L122

Reply:
This build browser demo have to be repeated since each of these job has its own VM. You build the demo in one VM but the publish is another. Also we do not need to build the demo anymore. The deploy script will automatically build the demo for you.

Reply:
@ltrung  The statement is contradicting itself. I thought the script might just be testing that browser demo is building correctly.

Reply:
@devalevenkatesh I followed this workflow as there are two jobs but one does not depend on other https://github.com/aws/amazon-chime-sdk-js/blob/SplitDeployCanaryDemo/.github/workflows/webview-ui-test.yml

Reply:
Also if I put needs these jobs will run in sequential order not parallely.

Reply:
Tested it out jobs do not need to depend on each other